Backpack-SwiftUI/Text/Classes/BPKText.swift

```diff
@@ -31,18 +31,26 @@ public struct BPKText: View {
     private var textColor = Color(BPKColor.textPrimaryColor)
     private var lineLimit: Int? = 1
     private var strikethrough: Bool = false
+    private var ignoreTextColorAndInheritFromParent: Bool = false
```

Comment:
I'm not sure how you feel about this @frugoman? But I assume this is why BPKChip used Text rather than BPKText. We were forcing a `textColor` to be set rather than letting BPKText inherit from its parent just like a regular Text would. At least that's how it seems to me